src/quo2/components/wallet/account_card/view.cljs

```diff
@@ -155,10 +172,11 @@
 (defn- view-internal
   [{:keys [type] :as props}]
   (case type
```

Comment:
pretty sure this could be:
```

  (case type
    (:watch-only :default :empty :missing-keypair) [user-account props]
    :add-account [add-account-view props]
    nil)
```

Reply:
Done